commands/copy.go

```diff
@@ -37,6 +38,7 @@ func NewCopyCommand(stdout, stderr io.Writer) *cobra.Command {
 	}
 	cmd.Flags().DurationVar(&r.timeout, "timeout", 5*time.Second, "Time limit for requests")
 	cmd.Flags().StringVarP(&r.password, "password", "p", "", "Password for encryption/decryption")
+	cmd.Flags().IntVar(&r.maxBufSize, "maxSize", 500, "Max data size in MB")
```

Comment:
`--max-size` is more preferable.
```suggestion
	cmd.Flags().IntVar(&r.maxBufSize, "max-size", 500, "Max data size in MB")
```